SeriesCruncher/app/src/main/java/com/balazs_csernai/seriescruncher/seriesdetails/model/episode/DescendingEpisodeComparator.java

```diff
@@ -0,0 +1,20 @@
+package com.balazs_csernai.seriescruncher.seriesdetails.model.episode;
+
+import java.util.Comparator;
+
+/**
+ * Created by erik_markus_kramli on 2016-02-08.
+ */
+public class DescendingEpisodeComparator implements Comparator<EpisodeModel> {
```

Comment:
Nice, but it could have been solved by extending DescendingEpisodeComparator with:
compare {return -super.compare;} ;)